src/hooks/useCreateQuizSkeleton.js

```diff
@@ -0,0 +1,39 @@
+import axios from "../config/axios";
+import api from "../config/api";
+import errorDisplayed from "../config/error";
+import useAuth from "./useAuth";
+
+export const useCreateQuizSkeleton = () => {
+  const { user } = useAuth();
+
+  const createQuizSkeleton = async (values, onError) => {
+    try {
+      const responseCreateQuiz = await axios.post(
+        api.quizzes,
+        JSON.stringify({ id_user_owner: user.userId,
+          title: values.title,
+          description: values.description,
+          categories: values.category }),
+        {
+          headers: { Authorization: `Bearer ${user.token}`,
+            "Content-Type": "application/json" },
+          withCredentials: true
+        }
+      );
+      console.log("responseCreateQuizSkeleton", responseCreateQuiz);
+    } catch (error) {
+      if (typeof onError === "function") {
+        if (!error.response) {
+          onError(errorDisplayed.server);
+        } else {
+          onError(error.response);
+        }
+      }
+    }
+  };
+  return {
+    createQuizSkeleton
+  };
+};
+
+export default useCreateQuizSkeleton;
```

Comment:
GG for the work! 👌 Just made some small comments

Reply:
Thanks :)))))))))